wemake_python_styleguide/patches/baseline.py

```diff
@@ -0,0 +1,71 @@
+from collections import defaultdict
+from typing import Iterable, Tuple, Type
+
+from flake8.checker import Manager
+
+from wemake_python_styleguide.logic import baseline
+
+
+def _patch_report(manager: Type[Manager]) -> None:
+    original_report = manager.report
+
+    def report(self) -> Tuple[int, int]:  # noqa: WPS430
+        # --- patch start
+        saved_reports: baseline.SavedReports = defaultdict(list)
+        self.saved_reports = saved_reports  # mypy requires that!
+        # --- patch end
+
+        report_result = original_report(self)
+
+        # --- patch start
+        if self.baseline is None:
+            self.baseline = baseline.save_to_file(self.saved_reports)
```

Comment:
Maybe some message should be logged that baseline file is missing and it is going to be created?

Reply:
Sure 👍